release-notes/release-2.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: February 13, 2024
+---
+
+# Features
+- PHP Azure Build Pipelines
+- [RUM Mobile - React Native](https://docs.middleware.io/rum/rum-react)
+- Spend Analysis charts
+- Trace correlation drill down from specific log messages
+- Price calculator that analyzes your bill based on the usage of different products
```

Comment:
Updated Settings -> Billing page with a simple flow and a pricing calculator to help predict spend.